docsets/SpigotMC/docset.json

```diff
@@ -1,13 +1,24 @@
 {
     "name": "SpigotMC",
-    "version": "1.10.2",
-    "archive": "SpigotMC.tgz",
+    "version": "1.12.2",
+    "archive": "SpigotMC-1.12.2.tgz",
```

Comment:
You need to add this file as well.

Reply:
I did, I just renamed the existing version to `Spigot-1.10.2.tgz` to match the naming convention of the new version.

Reply:
Your pull request doesn't contain any file at this path: `docsets/SpigotMC/SpigotMC-1.12.2.tgz`.

Reply:
Fixed! :)